### DISCUSSION TITLE
PKa value

### DISCUSSION THREAD

**Discussion by ibalshammari** [Open-Systems-Pharmacology/Forum#999] [Questions & Problems] [answered]
I've received an error in PK-Sim stating "pKa value should be greater than zero". Is there a way to adjust it such that a pKa slightly less than zero is accepted? "My compound is approximately -0.20"

**Comment by msevestre** [comment]
Moving this to the forum

**Reply by desaimrunal** [reply]
Sir,
I am stuck with an issue with respect to hike in concentration after IV and PO administration.
I have raised the issue in forum.
Waiting for reply.
Mrunal

**Reply by Aedginto** [reply]
This would be a strong acid. The pka is used in the tissue to plasma partition coefficient algorithms and there is no algorithm applicable for strong acids as these would not a class of molecule administered as they cause necrosis. As a result, you would need to include your own tissue: plasma partition coefficients from experimentation to have any hope of having something usable.

**Comment by StephanSchaller** [comment, marked as answer]
@ibalshammari , no, negative values are not allowed in PK-Sim (not deemed necessary).
As your pKa is way out of the physiological range, you might as well set it to zero.
